boltrouter/bolt_endpoints.go

```diff
@@ -73,7 +76,14 @@ func (br *BoltRouter) RefreshEndpoints(ctx context.Context) error {
 func (br *BoltRouter) getBoltEndpoints(ctx context.Context) (BoltEndpointsMap, error) {
 	// If running locally, we can't connect to quicksilver.
 	if br.config.Local {
-		return BoltEndpointsMap{}, nil
+		// For local run, return the BoltEndPoints built from localIP address and
+		// main & failover point to same port (9000).
+		endpoints := make(BoltEndpointsMap)
+		endpoints["main_write_endpoints"] = []string{fmt.Sprintf("%s:9000", br.config.LocalIpAddress)}
+		endpoints["failover_write_endpoints"] = []string{fmt.Sprintf("%s:9000", br.config.LocalIpAddress)}
+		endpoints["main_read_endpoints"] = []string{fmt.Sprintf("%s:9000", br.config.LocalIpAddress)}
+		endpoints["failover_read_endpoints"] = []string{fmt.Sprintf("%s:9000", br.config.LocalIpAddress)}
+		return endpoints, nil
```

Comment:
What happens if `br.config.Local` is true but the user did not pass in the `localipaddress`? We will return `:9000` - would a sensible default like `localhost` make sense for `localipaddress?`

Reply:
I think it's better to check if user pass' localipaddress, if not then just 'return BoltEndpointsMap{}'. bolt and SK run on separate container, hence we can not use localhost in this case.

Reply:
sounds good to me!